pkg/plugins/golang/v3/scaffolds/init.go

```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	// ControllerRuntimeVersion is the kubernetes-sigs/controller-runtime version to be used in the project
-	ControllerRuntimeVersion = "v0.12.2"
+	ControllerRuntimeVersion = "v0.13.0"
 	// ControllerToolsVersion is the kubernetes-sigs/controller-tools version to be used in the project
 	ControllerToolsVersion = "v0.9.2"
```

Comment:
@oscr could we also update the controller tools here to the latest version which uses k8s 1.25?
We need to ask for a new release, see: https://github.com/kubernetes-sigs/controller-tools
I am adding it to the channel now.